ansible/group_vars/all.yml

```diff
@@ -40,6 +40,18 @@ insecure_registrys:
 # The port that the Kubernetes apiserver component listens on.
 kube_master_api_port: 443
 
+# Required for HA master configuration
+# Kubernetes master apiserver interface
+#kube_apiserver_interface: eth1
+
+# Required for HA master configuration
+# A floating ip address for multiple kube-apiservers
+# Set to an address that is routeable from all nodes in the cluster
+#kube_apiserver_vip: 172.32.128.100
+
+# The hostname or IP address of the primary apiserver, or VIP for in an HA configuration.
+kube_apiserver_endpoint: "{% if groups['masters']|length > 1 %}{{ kube_apiserver_vip }}{% else %}{{ groups['masters'][0] }}{% endif %}"
```

Comment:
I don't believe this param should be exposed to a user. Can this be implemented in the defaults/main of the masters role?